pkg/workloads/workloads.go

```diff
@@ -198,7 +199,7 @@ func NewTimeSeriesReader(threadId int, threadCount int, pkCount int64, ckCount i
 	}
 	generator := rand.New(rand.NewSource(int64(time.Now().Nanosecond() * (threadId + 1))))
 	pkStride := int64(threadCount)
-	pkOffset := int64(threadId) % pkCount
+	pkOffset := (int64(threadId) % pkCount) + basicPkOffset
```

Comment:
`TimeSeriesWrite` iterates pk from `PkOffset` to `PkCount`, if you add `basicPkOffset` only to `PkOffset` if resulted number bigger or quels to `PkCount` it will return only `PkOffset` for primary key, which is a bug.
What you would might want is to add `basicPkOffset` to a `PkCount` shifting whole window by `basicPkOffset` not only it's lower bound.

Reply:
Done

Reply:
Line 223